Projects/Modules/NetworkModule/Sources/Interceptor/DefaultLoggingInterceptor.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import OSLog
+
+#if DEBUG
+
+public final class DefaultLoggingInterceptor: Interceptor {
+    
+    private let logger = Logger(subsystem: Bundle.main.bundleIdentifier ?? "", category: "NETWORK")
+        
+    public func willRequest(_ request: URLRequest, from endpoint: any Endpoint) throws {
+        guard let url = request.url else { throw NetworkError.invaildURL }
+        let method = endpoint.method
+        
+        var log = "====================\n\n[\(method)] \(url)\n\n====================\n"
+        log.append("✅ Endpoint: \(endpoint)\n")
+        
+        log.append( "------------------- Header -------------------\n")
+        if let header = endpoint.header, !header.isEmpty {
+            log.append("✅ Header: \(header)\n")
+        }
+        log.append( "------------------- Header END -------------------\n")
+        
+        if let body = request.httpBody, !body.isEmpty, let bodyString = String(bytes: body, encoding: .utf8) {
+            log.append("✅ Body: \(bodyString)\n")
+        }
+                
+        log.append("------------------- END \(method) --------------------------\n")
+        logger.log(level: .debug, "\(log)")
+    }
+    
+    public func didRecieve(_ result: Response, from endpoint: any Endpoint) throws {
+        
+        guard let httpResponse = result.response as? HTTPURLResponse else {
+            throw NetworkError.invaildResponse
+        }
+        
+        switch httpResponse.statusCode {
+        case endpoint.validationCode:
+            onSucceded(result, from: endpoint)
+            
+        default:
+            onFail(result, from: endpoint)
+        }
+        
+    }
+    
+}
```

Comment:
이부분 붙여주시면 좋을 것 같습니다! (코드 컨벤션) 아래 `extension`은 잘 붙여있는 것 같습니다.

```suggestion
        }
    }
}
```